packages/postcss-merge-longhand/src/lib/validateWsc.js

```diff
@@ -16,7 +16,7 @@ const styles = new Set([
 ]);
 
 function isStyle(value) {
-  return value && styles.has(value.toLowerCase());
+  return value !== undefined && styles.has(value.toLowerCase());
```

Comment:
Technically the old code is different as it would return `value` if value was falsy, but really the only case is `undefined`. I don't think it ever gets assigned to `null`.